attributes/default.rb

```diff
@@ -1,7 +1,9 @@
 # encoding: utf-8
 
-# This attributes is only taken into account for RHEL installations
-default['aurora']['version'] = '0.12.0-1'
+default['aurora']['version'] = case node['platform_family']
+    when 'rhel' then '0.12.0-1'
+    else ''
+end
```

Comment:
You removed the comment, it means that this attribute it still taken into account only on RHEL ?
If not, why do we don't converge to the same version i.e. 0.12.0 ?